packages/at_onboarding_cli/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM dart:2.19.4@sha256:5d2f42239d719bef7fe60abb37273aea16d8526980662b696c218663ac005d96 AS buildimage
+ENV HOMEDIR=/atsign
+ENV BINARYDIR=/atsign/at_activate
+ENV USER_ID=1024
+ENV GROUP_ID=1024
+WORKDIR /atsign
+# Context for this Dockerfile needs to be {at_libraries_repo}/packages/at_onboarding_cli
+# If building manually then (from packages/at_onboarding_cli):
+## docker build -t atsigncompany/at_activate .
+COPY . .
+RUN \
+  mkdir -p $HOMEDIR ; \
+  mkdir -p $BINARYDIR ; \
+  dart pub get ; \
+  dart pub upgrade ; \
+  dart compile exe bin/activate_cli.dart -o $BINARYDIR/at_activate ; \
+  tar cvzf /atsign/at_activate.`uname -m`.tar.gz at_activate/at_activate
```

Comment:
A few nits here:

* Backticks for shell commands are discouraged in style guides, so maybe use `"$(uname -m)"`
* We could simplify what's going to be a complex filename by having a `.tgz` suffix (as these days people aren't going to separately gunzip then untar).
* `at_activate/at_activate` assumes our working directory is /atsign (which it is) but we've got something of a random mix of `$DIR` and `/dir` and `dir` and it would be nice to have a more consistent style.

So altogether this line could (should?) be:
`tar cvzf /atsign/at_activate."$(uname -m)".tgz "$BINARYDIR"/at_activate`

Reply:
This is primarily for downloads and releases to go alongside sshnp which follows a similar pattern of tar.gz. with no support for windows.
Using docker buildx this docker file can create some , and in the future maybe all of them with darwin/ .
Doing this by hand is tedious so this helps a little.

Reply:
@cpswan I tried using the style guide "$uname -m)" and it does not work with or without the " so have gone back to backticks..

thanks for the other edits

Reply:
just a typo